cmd/registry/cmd/config/set.go

```diff
@@ -24,8 +24,16 @@ import (
 func setCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "set PROPERTY VALUE",
-		Short: "Set the value of a property.",
-		Args:  cobra.ExactArgs(2),
+		Short: "Set a property value in the active configuration.",
+		Long: "Set a property value in the active configuration. The following are valid properties:\n" +
+			"  - registry.address\n" +
+			"  - registry.insecure\n" +
+			"  - registry.location\n" +
+			"  - registry.project\n" +
+			"  - token-source",
+		Example: "registry config set registry.address localhost:8080\n" +
+			"registry config set token-source 'gcloud auth print-access-token email@example.com'",
+		Args: cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Does setting the registry.location and registry.project affect any commands?

Does it make `registry list project/project1/locations/global/apis` command shorter to `registry list apis` ?

Reply:
Yes, that's exactly what it does.

Reply:
I am going to give this a try.. Would be very useful. Does the `registry config configurations list` print these settings?

Reply:
Yes, when location and project are set, the "project/PROJECT_ID/location/LOCATION_ID" prefix gets added when needed.

Reply:
No, `registry config configurations list` is meant to be a simple table. You can see all properties using `registry config list` (for the active configuration) or `registry config configurations describe NAME` (for a specific configuration).